tests/unit/api/test_auth.py

```diff
@@ -56,7 +64,15 @@ def test_auth_missing_token():
 def test_auth_missing_config_public_key(monkeypatch):
     monkeypatch.delenv('JWT_PUBLIC_KEY')
     with pytest.raises(HTTPException) as e:
-        token = encode_jwt_payload(test_data.valid_jwt_payload, JWT_PRIVATE_KEY)
+        token = encode_jwt_payload(
+            test_data.valid_jwt_payload, JWT_PRIVATE_KEY
+        )
         authorize_user(f'Bearer {token}')
     assert e.value.status_code == 500
     assert e.value.detail == "Internal Server Error"
+
+
+def test_auth_toggled_off(monkeypatch):
```

Comment:
We could have one more test when TOGGLE_AUTH is not in the environment. This case will happen most of the time.

Reply:
There is not toggle auth env var set for all of the previous tests. It will then default to 'OFF'